action_type.cpp

```diff
@@ -2,6 +2,10 @@
 #include <cassert>
 #include <sstream>
 
+#ifdef HAS_MAGIC_ENUM // HAS_MAGIC_ENUM is defined in game.pro

```

Comment:
I enjoy the politeness of not making the game fail if members have not installed magic_enum yet :)